zeebe-test-runner/src/main/kotlin/io/zeebe/bpmnspec/runner/zeebe/ZeebeEnvironment.kt

```diff
@@ -10,13 +10,12 @@ import org.testcontainers.containers.GenericContainer
 import org.testcontainers.containers.Network
 import org.testcontainers.containers.wait.strategy.Wait
 
-class ZeebeEnvironment {
+class ZeebeEnvironment (
+        val zeebeImage : String = System.getProperty("zeebeImage", "camunda/zeebe-with-hazelcast-exporter"),
+        val zeebeImageVersion : String = System.getProperty("zeebeImageVersion", "0.24.2-0.10.0-alpha1")) {
```

Comment:
Currently, there is no way to pass these arguments in. The `ZeebeTestRunner` doesn't allow to configure the environment.